src/algebra/ordered_group.lean

```diff
@@ -93,482 +96,683 @@ instance ordered_comm_group.has_exists_mul_of_le (α : Type u)
+by rw [inv_le_div_iff_le_mul_left, mul_comm]
 
-lemma sub_le_iff_le_add : a - c ≤ b ↔ a ≤ b + c :=
-by rw [sub_le_iff_le_add', add_comm]
+@[to_additive]
+lemma div_le_iff_le_mul' : a / b ≤ c ↔ a ≤ b * c :=
+by rw [mul_comm, div_le_iff_le_mul]
 
-@[simp] lemma neg_le_sub_iff_le_add : -b ≤ a - c ↔ c ≤ a + b :=
-le_sub_iff_add_le.trans neg_add_le_iff_le_add'
+@[to_additive]
+lemma le_div_iff_mul_le' : b ≤ c / a ↔ a * b ≤ c :=
+by rw [mul_comm, le_div_iff_mul_le]
 
-lemma neg_le_sub_iff_le_add' : -a ≤ b - c ↔ c ≤ a + b :=
-by rw [neg_le_sub_iff_le_add, add_comm]
+end le_left
 
-lemma sub_le : a - b ≤ c ↔ a - c ≤ b :=
-sub_le_iff_le_add'.trans sub_le_iff_le_add.symm
+section lt_left
+variable [covariant_class α α (*) (<)]
 
-theorem le_sub : a ≤ b - c ↔ c ≤ b - a :=
-le_sub_iff_add_le'.trans le_sub_iff_add_le.symm
+@[to_additive]
+lemma lt_mul_of_inv_mul_lt_right (h : c⁻¹ * a < b) : a < b * c :=
+by { rw mul_comm, exact inv_mul_lt_iff''.mp h }
 
-lemma lt_sub_iff_add_lt' : b < c - a ↔ a + b < c :=
-by rw [sub_eq_add_neg, add_comm, lt_neg_add_iff_add_lt]
+@[to_additive]
+lemma inv_mul_lt_right_of_lt_mul (h : a < b * c) : c⁻¹ * a < b :=
+by { rw mul_comm at h, exact inv_mul_lt_iff''.mpr h }
 
-alias lt_sub_iff_add_lt' ↔ add_lt_of_lt_sub_left lt_sub_left_of_add_lt
+@[to_additive]
+lemma inv_lt_iff_one_lt_mul : a⁻¹ < b ↔ 1 < b * a :=
+by rw [inv_lt_iff_one_lt_mul', mul_comm]
 
-lemma lt_sub_iff_add_lt : a < c - b ↔ a + b < c :=
-by rw [lt_sub_iff_add_lt', add_comm]
+@[to_additive]
+lemma lt_inv_iff_mul_lt_one : a < b⁻¹ ↔ a * b < 1 :=
+by rw [lt_inv_iff_mul_lt_one', mul_comm]
 
-alias lt_sub_iff_add_lt ↔ add_lt_of_lt_sub_right lt_sub_right_of_add_lt
+@[to_additive]
+lemma inv_lt_self (h : 1 < a) : a⁻¹ < a :=
+(inv_lt_one_iff_one_lt.mpr h).trans h
 
-lemma sub_lt_iff_lt_add' : a - b < c ↔ a < b + c :=
-by rw [sub_eq_add_neg, add_comm, neg_add_lt_iff_lt_add]
+@[to_additive]
+lemma inv_mul_lt_iff_lt_mul_right : c⁻¹ * a < b ↔ a < b * c :=
+by rw [inv_mul_lt_iff'', mul_comm]
 
-alias sub_lt_iff_lt_add' ↔ lt_add_of_sub_left_lt sub_left_lt_of_lt_add
+@[to_additive]
+lemma div_lt : a / b < c ↔ a / c < b :=
+by rw [div_lt_left, mul_comm, div_eq_mul_inv]
 
-lemma sub_lt_iff_lt_add : a - c < b ↔ a < b + c :=
-by rw [sub_lt_iff_lt_add', add_comm]
+@[to_additive]
+theorem lt_div : a < b / c ↔ c < b / a :=
+by rw [lt_div_left, mul_comm, div_eq_mul_inv]
 
-alias sub_lt_iff_lt_add ↔ lt_add_of_sub_right_lt sub_right_lt_of_lt_add
+@[to_additive]
+lemma lt_div_iff_mul_lt' : b < c / a ↔ a * b < c :=
+by rw [mul_comm, lt_div_iff_mul_lt]
 
-@[simp] lemma neg_lt_sub_iff_lt_add : -b < a - c ↔ c < a + b :=
-lt_sub_iff_add_lt.trans neg_add_lt_iff_lt_add_right
+@[to_additive]
+lemma div_lt_iff_mul : c / a < b ↔ c < a * b :=
+by rw [mul_comm, div_lt_iff_lt_mul']
 
-lemma neg_lt_sub_iff_lt_add' : -a < b - c ↔ c < a + b :=
-by rw [neg_lt_sub_iff_lt_add, add_comm]
+@[simp, to_additive]
+lemma div_lt_iff_lt_mul : a / b < c ↔ a < b * c :=
+by rw [mul_comm, div_lt_iff_lt_mul']
```

Comment:
It looks to me like you've got the primes backwards on this one, which is why you've had to change `sub_lt_iff_lt_add` to `sub_lt_iff_lt_add'` throughout your patch. Can you swap the names around to make the diff smaller?

Reply:
I am trying to swap the names around, but I may have gotten into a little bit of a mess.  Let's see if CI builds... in a small number of iterations!

Reply:
You swapped all the names around without actually renaming this lemma, and now CI is tellling you to change each one back! You need to start by changing this line to:
```suggestion
@[simp, to_additive]
lemma div_lt_iff_lt_mul' : a / b < c ↔ a < b * c :=
by rw [mul_comm, div_lt_iff_lt_mul]
```

Reply:
Ah, I had missing this comment!  It took me much longer than it took you to figure out what was going on!